code-challenges.js

```diff
@@ -6,17 +6,47 @@
 
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in any two strings and returns the string with more characters. Use the two sets of test variables provided.
 
+// create a function called longerString
+//input: string1 and string2
+//create a conditional statement that evaluates the length of both strings. if string1 is greater then the statement is true.
+//if it's false then string 2 will be used
+//output: should be the string with more characters
+
+const longerString = (string1, string2) => {
+  if (string1.length > string2.length){
+    return string1
+  } else {
+    return string2
+  }
+}
 // Set one:
 const fruit1 = "apple"
 const fruit2 = "banana"
-
+// console.log(longerString(fruit1, fruit2))
 // Set two:
 const fruit3 = "cherry"
 const fruit4 = "kiwi"
-
-
+// console.log(longerString(fruit2, fruit3))
+// this question and previous ones i couldn't use the variable name because the nane wasn't initialize. What i did was input and pass the value of the variable instead of the name. I figured that if you were to change the position of the log then it would read the variable name. Has to do with either local and global scope. Glad that i found out.
```

Comment:
This looks great!  Oftentimes variables are declared before the function, however, as long as the variable is declared before calling on the function, javascript will read it.